src/QsCompiler/Transformations/ContentLifting.cs

```diff
@@ -439,7 +439,9 @@ public override ResolvedTypeKind OnTypeParameter(QsTypeParameter tp)
                     // Reroute a type parameter's origin to the newly generated operation
                     if (!this.SharedState.IsRecursiveIdentifier && this.SharedState.OldName.Equals(tp.Origin))
                     {
+#pragma warning disable 618 // QsTypeParameter.Range is obsolete.
```

Comment:
It doesn't have access to `ResolvedType.Range` because the parameter its given is only a `QsTypeParameter`. The update is non-trivial, so I will do it in a different PR.